src/manager.cpp

```diff
@@ -33,6 +34,9 @@ Manager::Manager(
         // Set up minimal things that is needed before bus name is claimed.
         m_worker->performInitialSetup();
 
+        // set callback to detect any asset tag change
+        registerAssetChangeCallback();
```

Comment:
better to add Tag also into the name registerAssetTagChangeCallback();

Reply:
Agree.

Reply:
are you going to change this also ?